releases/protocol/release-specs-v1.6.0.md

```diff
@@ -0,0 +1,712 @@
+- [#5170](https://github.com/multiversx/mx-chain-go/pull/5170) - Moved the WebSocketDriver (now named `HostDriver) in this repository. Integrated the FullDuplexHost from mx-chain-communication-go repository
+- [#5252](https://github.com/multiversx/mx-chain-go/pull/5252) - Updated the version of the mx-chain-es-indexer-go library
+- [#5265](https://github.com/multiversx/mx-chain-go/pull/5265) - Fixed the indexer creation
+- [#5278](https://github.com/multiversx/mx-chain-go/pull/5278) - Added new configuration file definitions
+- [#5286](https://github.com/multiversx/mx-chain-go/pull/5286) - Added a subscribe-call mechanism for the outport drivers to be able to request and get the current node settings
+- [#5384](https://github.com/multiversx/mx-chain-go/pull/5384) - Added the possibility to define a set of WebSocket host drivers instead of just one
+
+## 9. Sync missing trie nodes [#4616](https://github.com/multiversx/mx-chain-go/pull/4616)
+If there was a bug during processing and a trie node was missing (either because it was deleted or because it was never saved in the first place),
+that validator would have not been able to move forward with block processing.
+When a missing trie node is reached, sync it from the network. In this way, the network nodes will be able to repair themselves.
+
+### Impact:
+* No activation epoch
+* No configuration files changes
+* No binary flags changes
+* No API endpoints changes
+
+### Relevant PRs:
+- [#4588](https://github.com/multiversx/mx-chain-go/pull/4588) - Propagate the getNodeFromDB error until baseSync. This will automatically trigger the state sync
+- [#4724](https://github.com/multiversx/mx-chain-go/pull/4724) - Handle get nodes from db error in order to pass also the missing node hash
+- [#4830](https://github.com/multiversx/mx-chain-go/pull/4830) - Removed constants for peer and user accounts key and use constants from dataRetriever storage units
+- [#5188](https://github.com/multiversx/mx-chain-go/pull/5188) - Pass storageMarker to SyncAccounts() func. This way, a disabled storage marker can be given in case the db should not be marked as synced
+- [#5217](https://github.com/multiversx/mx-chain-go/pull/5217) - Added a notifier in blockchainHook which notifies the state syncer that a data trie node is missing. This way the state syncer can start to sync the missing data trie nodes
+- [#5258](https://github.com/multiversx/mx-chain-go/pull/5258) - Get identifier from trieStorageManager and not from storer
+- [#5271](https://github.com/multiversx/mx-chain-go/pull/5271) - Fixed failing unit test
+- [#5273](https://github.com/multiversx/mx-chain-go/pull/5273) - Proper releases for libraries
+
+## 10. Balance data tries [#4636](https://github.com/multiversx/mx-chain-go/pull/4636)
+
+The keys at which values are saved in a data trie are not random. Because of this, the data tries are not balanced, thus resulting in more intermediary nodes.
```

Comment:
The keys where the values are saved in a data trie are not random

Reply:
done